app/ui.js

```diff
@@ -319,7 +319,8 @@ const UI = {
     addConnectionControlHandlers() {
         document.getElementById("noVNC_disconnect_button")
             .addEventListener('click', UI.disconnect);
-        document.getElementById("noVNC_connect_button")
+        document.getElementById("noVNC_connect_box")
+            .getElementsByTagName('button')[0]
```

Comment:
Finding specific elements should really be done based on id. Both for clarity and robustness.

Reply:
Fixed in 05baf14256a4640e15cda63cc2ac576f44482775